### PR TITLE
Fix nanoid and revert default VSIX version

### PR DESCRIPTION
**Pull request by MartinBalin**
Fix nanoid npm version to 3.3.8 due to https://github.com/advisories/GHSA-mwcw-c2x4-8c55 keeping mocha version the same. Revert back default VSIX version to 0.1.0.

### CODE REVIEW

**Review by sdedic — APPROVED** [review]

**Comment by sdedic** [comment]
@mbien could you please help me understanding why the paperwork failed ? The error message reads:
```
Error: PRs must be labeled, see: https://cwiki.apache.org/confluence/display/NETBEANS/PRs+and+You+-+A+reviewer+Guide
```
and as I can see the script checks that at least one label is present on the PR. This PR was labelled 'VSCode Extension' before the tests were re-run today. What went wrong ?

**Comment by MartinBalin** [comment]
This is the case I usually add labels when PR is created.

**Comment by mbien** [comment]
> and as I can see the script checks that at least one label is present on the PR. This PR was labelled 'VSCode Extension' before the tests were re-run today. What went wrong ?

@sdedic this [section on the guide](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=240884239#PRsandYouAreviewerGuide-Important:) describes the detail

If the labels are changed you need a fresh workflow run. This can be done via a regular PR sync event (branch changes). The lock/unlock trick was added for situations when reviewers have to change labels of non-committers. But it ended up being also useful for reviewers which wanted to add more jobs to the PR without having to ping someone.

If everything was labeled before the PR was created then this might have been a github bug - I don't know. Please check that the right jobs run right before pressing merge! If something doesn't look right - do a rebase and it will also start a fresh workflow run.

**Comment by mbien** [comment]
Btw checking that the right jobs ran is part of our approval process too.

approval and merge check lists:
https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=240884239#PRsandYouAreviewerGuide-PRapprovalchecklist

**Pull request merged**
